fedramp_low_rev_4/fedramp_low_rev_4.sp

```diff
@@ -0,0 +1,24 @@
+locals {
+  fedramp_low_rev_4_common_tags = merge(local.aws_compliance_common_tags, {
+    fedramp_low_rev_4 = "true"
+    type              = "Benchmark"
+  })
+}
+
+benchmark "fedramp_low_rev_4" {
+  title         = "Federal Risk and Authorization Management Program (FedRAMP - Low)"
+  description   = "The Federal Risk and Authorization Management Program (FedRAMP) is a US government-wide program that delivers a standard approach to the security assessment, authorization, and continuous monitoring for cloud products and services."
+  children = [
```

Comment:
Are we able to add any documentation for this top level benchmark (which should also help users understand what FedRAMP Low is compared to FedRAMP Moderate)?